### DISCUSSION TITLE
Ember Travel Mug 2

### DISCUSSION THREAD

**Discussion by z-master42** [sopelj/hass-ember-mug-component#33] [General]
I was able to link my Travel Mug 2. Unlike my Cup, it didn't show up via auto discovery before. I had to connect and pair it via bluetoothctl, but that went smoothly, then I also had a notification in Home Assistant that a new device had been discovered.

Unfortunately I was too stupid to get the info from the console, so only a screenshot.

![grafik](https://github.com/sopelj/hass-ember-mug-component/assets/66380371/5d03fc14-8db6-485e-ab58-329292f694d3)

As announced [here](https://github.com/sopelj/hass-ember-mug-component/issues/31#issuecomment-1614661497), you can now complete the doc.

**Comment by sopelj** [comment]
Oh thanks! That's weird that it wasn't detected as the service UUID is the same. I added the name to the list of device names to be discovered as of 0.8.0-beta-2. Hopefully that will work for it. If you have a chance to test it. Thanks!
